app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -149,6 +204,63 @@ class PoliciesActivityTest {
       }
     }
   }
+  @Test
+  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
+    launch<PoliciesActivity>(
+      PoliciesActivity.createPoliciesActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { activityScenario ->
+      activityScenario.onActivity { activity ->
+
+        testCoroutineDispatchers.runCurrent()
+        val htmlParser = htmlParserFactory.create(
+          policyOppiaTagActionListener = mockPolicyOppiaTagActionListener,
+          displayLocale = appLanguageLocaleHandler.getDisplayLocale()
+        )
+        val textView: TextView =
+          activity.findViewById(R.id.policy_description_text_view)
+
+        val htmlResult: Spannable = htmlParser.parseOppiaHtml(
```

Comment:
Doesn't the fragment already do this part--why do we need to parse it again?

Reply:
To perform click on the link I used html parser here. I have removed is as per https://github.com/oppia/oppia-android/pull/4595#issuecomment-1276766636